src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -6316,6 +6375,27 @@ export class StudyViewPageStore
             );
         }
 
+        if (this.shouldDisplaySampleTreatmentTarget.result) {
```

Comment:
is it weird that you test just for the sample (shouldDisplaySampleTreatmentTarget) but then change viseability for both sample and patient?